en_US/LC_MESSAGES/django.po

```diff
@@ -8718,71 +8669,164 @@ msgstr "An Error Occurred"
 msgid "Find help..."
 msgstr "Find help..."
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:24
+#: kitsune/sumo/jinja2/includes/common_macros.html:43 kitsune/sumo/jinja2/includes/common_macros.html:92 kitsune/sumo/jinja2/includes/common_macros.html:158
 #, fuzzy
-msgid "Get Community Support"
-msgstr "Community Store"
+msgid "Close Firefox menu"
+msgstr "Firefox"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:50
+msgid "Create a thread and find answers by posting a question to any of our product support forums."
+msgstr "Create a thread and find answers by posting a question to any of our product support forums."
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:26
+#: kitsune/sumo/jinja2/includes/common_macros.html:54 kitsune/sumo/jinja2/includes/common_macros.html:73
 #, fuzzy
-msgid "Contributor Tools"
-msgstr "Contributor Dashboard"
+msgid "Firefox Browser"
+msgstr "Firefox"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:33
-msgid "Volunteer"
-msgstr "Volunteer"
+#: kitsune/sumo/jinja2/includes/common_macros.html:59
+#, fuzzy
+msgid "Browse All Product Forums"
+msgstr "Support Forum"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:70
+#: kitsune/sumo/jinja2/includes/common_macros.html:67
 #, fuzzy
-msgid "For Contributors"
-msgstr "How to Contribute"
+msgid "Explore Our Help Articles"
+msgstr "All Articles"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:82
+#: kitsune/sumo/jinja2/includes/common_macros.html:69
+msgid "Dig into the knowledge base, tips and tricks, troubleshooting, and so much more."
+msgstr "Dig into the knowledge base, tips and tricks, troubleshooting, and so much more."
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:78
 #, fuzzy
-msgid "Quickstart Guide"
-msgstr "Contributor Dashboard"
+msgid "See All Articles"
+msgstr "All Articles"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:83
+#: kitsune/sumo/jinja2/includes/common_macros.html:88
 #, fuzzy
-msgid "News &amp; Resources"
+msgid "Contributor Tools"
 msgstr "Contributor Dashboard"
 
-#: kitsune/sumo/jinja2/includes/common_macros.html:84
+#: kitsune/sumo/jinja2/includes/common_macros.html:97
 #, fuzzy
-msgid "Contributor Forums"
-msgstr "Contributor Dashboard"
+msgid "Discussions"
+msgstr "Discussion"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:99
+msgid "Meet the community: read announcements, follow discussions and ask questions."
+msgstr "Meet the community: read announcements, follow discussions and ask questions."
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:111
+msgid "Tools"
+msgstr "Tools"
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:113
+msgid "Explore resources to help you with your contributions."
+msgstr "Explore resources to help you with your contributions."
+
+#: kitsune/sumo/jinja2/includes/common_macros.html:140 kitsune/sumo/jinja2/includes/common_macros.html:318 kitsune/wiki/jinja2/wiki/document.html:79
+#, fuzzy
+msgid "Download Firefox"
+msgstr "MozillaZine"
```

Comment:
I assume this will remain in English.

Reply:
Probably this needs change on the localization site. It's already marked as fuzzy.

Reply:
I believe this needs change int he msgstr string too.